parametric/spec/tracecontext.py

```diff
@@ -0,0 +1,126 @@
+# The Tracestate type is imported from https://github.com/w3c/trace-context/blob/84b583d86ecb7005a9eab8fed86ab7117b050b48/test/test.py,
```

Comment:
I'm not sure if this license allows us to include the source code or not. The other OpenTelemetry SDK repos do a `git clone` on the `w3c/tracecontext` repo and run these tests by implementing an HTTP server. That can be a fallback if we are unable to include the source in this repo.

Reply:
This is resolved now after receiving guidance from colleagues